gui/src/shared/components/kg/search/KgSourceLink.tsx

```diff
@@ -4,18 +4,20 @@ import * as React from "react";
 import {CSSProperties} from "@material-ui/core/styles/withStyles";
 import {kgId} from "shared/api/kgId";
 
-export const KgDatasourceLink: React.FunctionComponent<{
-  datasource: string;
+export const KgSourceLink: React.FunctionComponent<{
+  source: string | string[];
   style?: CSSProperties;
-}> = ({datasource, style}) => (
+}> = ({source, style}) => (
   <Link
     data-cy="datasource-link"
     style={style}
     to={Hrefs.kg({id: kgId}).nodeSearch({
-      filters: {sources: {include: [datasource]}},
+      filters: {
+        sources: {include: typeof source === "string" ? [source] : source},
+      },
       __typename: "KgNodeSearchVariables",
     })}
   >
-    {datasource}
+    {source}
```

Comment:
Won't work if source is a string[]? Probably source should just be a string, one link per source. The "combined" data sources (post merge) will be `string`.

Reply:
Hm thats true, I missed that. I will change source to be a string.